src/index.ts

```diff
@@ -16,6 +16,58 @@ const rl = createInterface({
   output: process.stdout
 })
 
+function connectToBootnode (node: any, bootNodeAddr: string) {
+  return new Promise((resolve, reject) => {
+    async.parallel(
```

Comment:
Here I would much rather if you used `Promise.race`. Some pattern like [this](https://italonascimento.github.io/applying-a-timeout-to-your-promises/) but I am actually sure there is something lower level that does timeouts on promises.

Reply:
Yes, this is what I meant about to remove the _async_ library, I will try to use native functions instead 👌🏻